qds_sdk/clusterv2.py

```diff
@@ -316,6 +318,8 @@ def set_cluster_info(self,
 
                 `datadog_app_token` : Specify the Datadog APP token to use the Datadog monitoring service
 
+                `notifications` : Specify the list of notification channels
```

Comment:
notification channel ids

Reply:
should the parameter me `notifications _channels` instead?